zulipterminal/model.py

```diff
@@ -1011,11 +1018,21 @@ def _handle_update_message_event(self, event: Event) -> None:
             # 'subject' is not present in update event if
             # the event didn't have a 'subject' update.
             if 'subject' in event:
+                old_subject = self.index['messages'][message_id]['subject']
                 new_subject = event['subject']
+                print("old subject =>", old_subject, flush=True)
+                print("new subject =>", new_subject, flush=True)
                 for msg_id in event['message_ids']:
                     self.index['messages'][msg_id]['subject'] = new_subject
                     self._update_rendered_view(msg_id)
 
+                view = self.controller.view
+                print("event=>", event, flush=True)
+                if (len(self.narrow) == 2
+                    and self.narrow[1][0] == 'topic'
+                        and old_subject != new_subject):
+                    view.set_footer_text("Subject changed", 3)
```

Comment:
No, it's for the same issue. This was for the case when a message's subject is edited outside of the current narrow. So the message disappears from the current narrow and is shifted to some other narrow. But, I think the logic behind this is also incorrect. I'll have to think more on this. Any pointers that you would like to give?

Reply:
This is sort of the same issue, but I think it would be fine to first work on showing a message when sending to a different narrow than the current one, then work on what happens with editing as a follow-up.

I think in both cases the focus is on what the active user does within this ZT session, so it's likely best to hook into the messages to the server, rather than events from it.

Reply:
I have moved the code for footer notification in case of a message update, to hook to the part where the request to the server is sent. Can you please review my PR whenever you get the time?